micromasters/urls.py

```diff
@@ -21,6 +22,7 @@
     url('', include('social.apps.django_app.urls', namespace='social')),
     url(r'^admin/', include(admin.site.urls)),
     url(r'^api/v0/', include(router.urls)),
+    url(r'^api/v0/dashboard/?$', UserDashboard.as_view(), name='dashboard_api'),
```

Comment:
Can you remove the `?` from the regex? Django should redirect it automatically if the trailing slash is not present